cypress/integration/checkout/checkout.multipiece.items.spec.js

```diff
@@ -0,0 +1,152 @@
+            cy.getHoldingSources({ limit: 2 });
+            cy.getInstanceTypes({ limit: 1 });
+            cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+              .then((users) => {
+                userBarcode = users[0].barcode;
+              });
+          })
+          .then(() => {
+            cy.createInstance({
+              instance: {
+                instanceTypeId: Cypress.env('instanceTypes')[0].id,
+                title: instanceTitle,
+              },
+              holdings: [{
+                holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
+                permanentLocationId: Cypress.env('locations')[0].id,
+                sourceId: Cypress.env('holdingSources')[0].id,
+              }],
+              items: [
+                [
+                  {
+                    barcode: FIRST_ITEM_BARCODE,
+                    numberOfPieces: '1',
+                    status: { name: 'Available' },
+                    descriptionOfPieces: descriptionOfPiece,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                  {
+                    barcode: SECOND_ITEM_BARCODE,
+                    numberOfPieces: quantityPiecesForSecondItem,
+                    status: { name: 'Available' },
+                    descriptionOfPieces: descriptionOfPiece,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                  {
+                    barcode: THIRD_ITEM_BARCODE,
+                    numberOfPieces: quantityPiecesForThirdItem,
+                    status: { name: 'Available' },
+                    descriptionOfPieces: descriptionOfPiece,
+                    numberOfMissingPieces: quantityOfMissingPieces,
+                    missingPieces: missingPieceDescription,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                  {
+                    barcode: FOURTH_ITEM_BARCODE,
+                    numberOfPieces: quantityPiecesForFourthItem,
+                    status: { name: 'Available' },
+                    numberOfMissingPieces: quantityOfMissingPieces,
+                    missingPieces: missingPieceDescription,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                ],
+              ],
+            });
+          });
+      });
+  });
+
+  after('Delete all data', () => {
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${FIRST_ITEM_BARCODE}"` })
```

Comment:
remember required instance properties into the test and use direct delete instead of extra getInstance request

Reply:
Done